### PR TITLE
Modify aws cluster config to expose variables to user

### PR DESCRIPTION
**Pull request by aloganat**
Signed-off-by: Arthy Loganathan <aloganat@redhat.com>

### CODE REVIEW

**Review by tarukumar — APPROVED** [review]
LGTM. Lets wait for the @pablofelix approval before we merge it

File: tasks/Resources/Provisioning/Hive/AWS/aws-cluster.yaml
Comment (jgarciao) [inline tasks/Resources/Provisioning/Hive/AWS/aws-cluster.yaml:80]: @pablofelix Is there a way to populate the same variable in two places?

**Review by pablofelix — CHANGES_REQUESTED** [review]
Variable cluster_image_set should have the same value as the base_image_set. It could be better to use only one variable image_set for example instead

**Comment by aloganat** [comment]
> 

Hi @pablofelix , As I have mentioned in the call, if I use the same variable for example image_set, its not been substituted in both the places - https://github.com/aloganat/ods-ci/blob/aws_config_hive/tasks/Resources/Provisioning/Hive/AWS/oc-provision.robot#L8 . Could you let me know if there is a way to do it?

**Review by pablofelix — APPROVED** [review]
lgtm

**Review by jgarciao — APPROVED** [review]

**Pull request merged**
